DQM/SiPixelPhase1Common/python/HistogramManager_cfi.py

```diff
@@ -313,6 +313,26 @@
 ]
 
 StandardSpecificationTrend_Num = [
+    Specification(IsOffline).groupBy("PXBarrel/Lumisection")
+                          .reduce("MEAN")
+                  .groupBy("PXBarrel","EXTEND_X")
+                  .save(),
+    Specification(IsOffline).groupBy("PXBarrel/Lumiblock")
```

Comment:
is this `Lumiblock` correct?

Reply:
@mmusich @arossi83 Sorry about this, this was leftover from testing, the current commit completely removes every change to SiPixelPhase1Common.